audb/core/dependencies.py

```diff
@@ -10,14 +10,51 @@
 
 
 class Dependencies:
-    r"""Hold dependencies of a database.
+    r"""Dependencies of a database.
+
+    :class:`audb.Dependencies` gathers all files
+    a database contains
+    and metadata about them
+    in a single object.
+    The metadata contain information
```

Comment:
I'm not sure, but does it have to be "contains"?

Reply:
Yes, you are right, I changed it accordingly.